specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -86,11 +95,167 @@
+          "type": "string"
+        },
+        "storageDiagnostics": {
+          "description": "For service use only. Diagnostic data occasionally included by the Azure Storage service. This property should be ignored by event consumers.",
+          "type": "object"
+        }
+      }
+    },
+    "StorageDirectoryDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Storage.DirectoryDeleted event.",
+      "type": "object",
+      "properties": {
+        "api": {
+          "description": "The name of the API/operation that triggered this event.",
+          "type": "string"
+        },
+        "clientRequestId": {
+          "description": "A request id provided by the client of the storage API operation that triggered this event.",
+          "type": "string"
+        },
+        "requestId": {
+          "description": "The request id generated by the storage service for the storage API operation that triggered this event.",
+          "type": "string"
+        },
+        "url": {
+          "description": "The path to the directory deleted.",
+          "type": "string"
+        },
+        "recursive": {
+          "description": "Is this event for a recursive delete operation.",
+          "type": "string"
+        },
+        "sequencer": {
+          "description": "An opaque string value representing the logical sequence of events for any particular directory name. Users can use standard string comparison to understand the relative sequence of two events on the same directory name.",
+          "type": "string"
+        },
+        "identity": {
+          "description": "The identity of the requester that triggered this event.",
+          "type": "string"
+        },
+        "storageDiagnostics": {
+          "description": "For service use only. Diagnostic data occasionally included by the Azure Storage service. This property should be ignored by event consumers.",
+          "type": "object"
+        }
+      }
+    },
+    "StorageBlobRenamedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Storage.BlobRenamed event.",
+      "type": "object",
+      "properties": {
+        "api": {
+          "description": "The name of the API/operation that triggered this event.",
+          "type": "string"
+        },
+        "clientRequestId": {
+          "description": "A request id provided by the client of the storage API operation that triggered this event.",
+          "type": "string"
+        },
+        "requestId": {
+          "description": "The request id generated by the storage service for the storage API operation that triggered this event.",
+          "type": "string"
+        },
+        "sourceUrl": {
+          "description": "The path to the blob that was renamed.",
+          "type": "string"
+        },
+        "destinationUrl": {
+          "description": "The path to the blob that was rename to.",
```

Comment:
Nit: Might want to reword this to "The new path to the blob after the rename operation".